util/config/config.go

```diff
@@ -128,7 +128,7 @@ var (
 	}
 	MaxTxnAttributesLen  = 100
 	AllowTxnRegisterName = HeightDependentBool{
-		heights: []uint32{7500, 0},
+		heights: []uint32{1111117500, 0},
```

Comment:
We shouldn't modify the existing value, but should prepend a new value to it, like `[]uint32{xxx, 7500, 0}`

Reply:
values